src/target/riscv/riscv-013.c

```diff
@@ -2811,6 +2818,320 @@ void riscv013_fill_dmi_nop_u64(struct target *target, char *buf)
 	buf_set_u64((unsigned char *)buf, DTM_DMI_ADDRESS_OFFSET, info->abits, 0);
 }
 
+static int get_max_sbaccess(struct target *target)
```

Comment:
Can you bracket this with some sort of comments that this is all related to the SBA test command?